src/WCSimConstructMaterials.cc

```diff
@@ -803,7 +803,7 @@ void WCSimDetectorConstruction::ConstructMaterials()
 
    OpWaterTySurface->SetType(dielectric_dielectric);
    OpWaterTySurface->SetModel(unified); 
-   OpWaterTySurface->SetFinish(groundbackpainted); //a guess, but seems to work
+   // OpWaterTySurface->SetFinish(groundbackpainted); //a guess, but seems to work
```

Comment:
Can you comment a bit more on this bug in the tyvek properties that you found?

Reply:
Unfortunately I haven't found that much information on the bug. Perhaps someone already familiar with material optical properties in Geant4 could enlighten us? Regarding for the OD, it doesn't really matter because I will submit in the next PR the WLS plates behind the PMTs. I checked by varying the different options for optical materials surface and couldnt see a difference whatsoever for light leaking between ID/OD.

Reply:
But, how do you know there is a bug? What kind of error were you getting or what kind of behaviour were you seeing?

Reply:
I get a seg fault during execution, and it seems to execute now on my WLS plates branches.